helm/bits/templates/bits.yaml

```
@@ -65,10 +65,16 @@ spec:
         - name: bits-assets
           mountPath: /assets/
         {{ end }}
+        {{- if or .Values.global.resources.cpu .Values.global.resources.memory }}
         resources:
           requests:
-            cpu: 800m
-            memory: 150Mi
+          {{- if .Values.global.resources.cpu }}
+            cpu: {{ .Values.global.resources.cpu }}
+          {{ end }}
```

```suggestion
            {{- with .Values.global.resources.cpu }}
            cpu: {{ . }}
            {{ end }}
```

(`with` is skipped if the value is not given.)

---

Done with commit 935c4a4f960b8b455d6c76fb727c1a65cab05f6d.